src/seedu/addressbook/data/person/Unit.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's unit in the address book.
+ */
+public class Unit {
+	
+	private String unit;
+	
+	public Unit(String unit_Num){
+		this.unit = unit_Num;
+	}
+	
+	public String getUnit(){
+		return this.unit;
+	}
+	
+	public void setUnit(String unit_Num){
+		this.unit = unit_Num;
+	}
+
+}
```

Comment:
You have not updated the test nor the documents to reflect these new changes. These need to change too.